lib/protocol/bep_extensions.go

```diff
@@ -213,6 +213,17 @@ func (f FileInfo) isEquivalent(other FileInfo, ignorePerms bool, ignoreBlocks bo
 	return false
 }
 
+func ModTimeEqual(a, b time.Time, modTimeWindowS int) bool {
+	if a == b {
+		return true
+	}
+	diff := a.Sub(b)
+	if diff < 0 {
+		diff *= -1
+	}
+	return diff < time.Duration(modTimeWindowS)*time.Second
```

Comment:
Should this be <=? Or we assume a single step change is large enough?
Not implying it should be that way, just opening up a discussion.

Reply:
I'd expect it to be, i.e. I would think that anything that happens in [0,2) to get a timestamp 0 and in [2, 4) to get 2 and so on. However that's just hopeful thinking on my part, I have no clue what FAT/android caching does in reality.